### PR TITLE
Update Broadcom AFBR API to 1.4.4

### PR DESCRIPTION
**Pull request by AlexKlimaj**
https://broadcom.github.io/AFBR-S50-API/migration_guide.html

This is a 250 quad with ARKV6X, ARK GPS, and ARK Flow. Very windy day.

https://review.px4.io/plot_app?log=1263490b-aecf-420e-af44-b1013802bd0a
https://review.px4.io/plot_app?log=6be883f2-33c3-4ff6-bf35-94cdf3981903

650 quad with ARKV6X, Dual ARK RTK GPS, ARK Flow, and a dronecan smart battery. Moving baseline for yaw. Very windy.

https://review.px4.io/plot_app?log=95943f96-4039-4b80-ad57-92311a29e532
https://review.px4.io/plot_app?log=20877cb1-6949-4813-8e25-30ce7d3a8d48

Different 250 quad with Pixhawk 4 Mini, ARK GPS, and ARK Flow Rev 3 with PAA3905 and AFBR_S50LX85D. Very windy day.

https://review.px4.io/plot_app?log=0b1a730d-147b-47e3-b25e-c56fe27dacf8
https://review.px4.io/plot_app?log=f353442a-f324-4b15-b4cb-109403e36ebe

Optical Flow fusion with GPS fusion on
https://review.px4.io/plot_app?log=d6a23cd7-a292-49c5-a543-cc483856d019

Optical Flow fusion with GPS fusion off
https://review.px4.io/plot_app?log=92315c6d-ed04-4e06-b534-995f269ffd85

### CODE REVIEW

**Comment by dirksavage88** [comment]
Can confirm the AFBRS50 is working with the new API update

**Comment by dirksavage88** [comment]
@AlexKlimaj is there plans to get this upstream or is there another version update in the works? There could be other can flow users that faced the same issues with the broadcom rangefinder not reporting valid ranges when using the old API version.

**Comment by AlexKlimaj** [comment]
@dirksavage88 @dagar This should be ready for merge.

File: src/drivers/uavcan/sensors/rangefinder.cpp [hunk 1]
```diff
@@ -108,14 +108,13 @@ void UavcanRangefinderBridge::range_sub_cb(const
...
-	 * FIXME HACK
-	 * This code used to rely on msg.getMonotonicTimestamp().toUSec() instead of HRT.
-	 * It stopped working when the time sync feature has been introduced, because it caused libuavcan
-	 * to use an independent time source (based on hardware TIM5) instead of HRT.
-	 * The proper solution is to be developed.
-	 */
-	rangefinder->update(hrt_absolute_time(), msg.range);
+	int8_t quality = -1;
+
+	if (msg.reading_type == uavcan::equipment::range_sensor::Measurement::READING_TYPE_VALID_RANGE) {
```
Comment (dagar) [inline src/drivers/uavcan/sensors/rangefinder.cpp:113]: Seems like we could use more the enum? https://github.com/dronecan/DSDL/blob/f618adf88990829abd152b48a7bbdfd58881a5a6/uavcan/equipment/range_sensor/1050.Measurement.uavcan#L19-L22

Comment (dagar) [reply src/drivers/uavcan/sensors/rangefinder.cpp:113]: For example we could use READING_TYPE_TOO_CLOSE and READING_TYPE_TOO_FAR properly, then lift or remove UAVCAN_RNG_MIN/UAVCAN_RNG_MAX so they don't need to be manually configured.

Comment (AlexKlimaj) [reply src/drivers/uavcan/sensors/rangefinder.cpp:113]: I'm not sure we have the ability to tell too close or too far from all the distance sensor drivers.

**Comment by AlexKlimaj** [comment]
Rebased on main and force pushed.

**Review by dagar — APPROVED** [review]

**Pull request merged**